docs/guides/3-javascript.md

```diff
@@ -1,28 +1,39 @@
 # Spectral in JavaScript
 
-The Spectral CLI is a thin wrapper around a JavaScript (TypeScript) API, which can be used independently to do all the same things outside of the CLI.
+The Spectral CLI is a thin wrapper around a JavaScript (TypeScript) API, which can be used independently to do all the same things outside of the CLI, such as linting YAML and JSON documents from a string or an object.
 
-Assuming it has been installed as a Node module via NPM/Yarn, it can be used to lint YAML and JSON documents from a string, or from an object.
+## Prerequisites
 
-First of all, in order to consume the JS API you need to install the appropriate package(s).
+To use the Spectral JS API, you need to install the appropriate package.
+
+For npm users:
 
 ```bash
 npm install -g @stoplight/spectral-core
 ```
 
-If you are a Yarn user:
+For Yarn users:
 
 ```bash
 yarn global add @stoplight/spectral-core
 ```
 
-## Linting a YAML String
+## Get Started
+
+Similar to using Spectral in the CLI, there are two things you'll need to run Spectral in JS:
+
+- A string or a file representing an API specification
```

Comment:
Hey hey, something we've tried to do in the past is be consistent on naming things, as OpenAPI has a million different words to mean the same thing: https://github.com/openapi-contrib/glossary 

```suggestion
- A string or a document containing your structured data (OpenAPI, AsyncAPI, Kubernetes, etc).
```

Reply:
This is really helpful, and makes total sense! I'll try to use those terms and check that glossary when I'm making changes in the future. :)